### PR TITLE
LR-134. Authentication

### PR DESCRIPTION
**Pull request by anatoliiibrahimov**
- [x] @vovka 
- [x] @brouhaha88 
- [x] @superNazo

### CODE REVIEW

**Comment by anatoliiibrahimov** [comment]
- [] @vovka 
- [] @brouhaha88 
- [] @superNazo

File: config/initializers/devise.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  config.strip_whitespace_keys = [:email]
+  config.skip_session_storage = [:http_auth]
+  config.stretches = Rails.env.test? ? 1 : 10
+  config.reconfirmable = true
+  config.expire_all_remember_me_on_sign_out = true
+  config.password_length = 8..72
+  config.timeout_in = 30.minutes
+  config.reset_password_within = 6.hours
+  config.sign_out_via = :delete
+  config.secret_key = Rails.application.secrets.secret_key_base || '8ed23f0f9f'
```
Comment (brouhaha88) [inline config/initializers/devise.rb]: Ignore

File: config/initializers/devise.rb [hunk 2]
```diff
@@ -0,0 +1,15 @@
+Devise.setup do |config|
+  config.omniauth :facebook, Rails.application.secrets.facebook_app_id, Rails.application.secrets.facebook_app_secret,
+                  callback_url: 'http://localhost:3000/users/auth/facebook/callback'
+  require 'devise/orm/active_record'
```
Comment (brouhaha88) [inline config/initializers/devise.rb]: Ignore

File: config/initializers/devise.rb [hunk 3]
```diff
@@ -0,0 +1,15 @@
+Devise.setup do |config|
+  config.omniauth :facebook, Rails.application.secrets.facebook_app_id, Rails.application.secrets.facebook_app_secret,
+                  callback_url: 'http://localhost:3000/users/auth/facebook/callback'
```
Comment (brouhaha88) [inline config/initializers/devise.rb]: ignore

File: db/schema.rb [hunk 4]
```diff
@@ -11,21 +11,34 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20160317123733) do
+ActiveRecord::Schema.define(version: 20160317123716) do
 
-  create_table "questions", force: :cascade do |t|
```
Comment (vovka) [inline db/schema.rb]: You removed questions table.

Comment (anatoliiibrahimov) [inline db/schema.rb]: now it is ok

**Comment by vovka** [comment]
Looks ood

**Pull request merged**